tests/fullstack-test2/dml/test_table_scan.test

```diff
@@ -0,0 +1,390 @@
+# Copyright 2022 PingCAP, Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TINYINT
```

Comment:
Can we make this test run with fewer rounds of create-wait-drop? It take serval tens of seconds to wait each tiflash replica to be available and will slow down the CI process...

Reply:
But I want to test the primary key of different data types...

Reply:
after "optimization", the cost time from 52s to 17s :)

Reply:
> But I want to test the primary key of different data types...

Maybe also create a clustered index table for testing common handle?